rust-toolchain.toml

```diff
@@ -1,3 +1,11 @@
 [toolchain]
-channel = "1.68.2"
-targets = [ "x86_64-unknown-linux-gnu" ]
+channel = "nightly-2024-09-05"
```

Comment:
Why nightly?

Reply:
This ensures that we get access to all necessary features of the Rust tools. Unfortunately, many useful features are only accessible in nightly (e.g. for `rustfmt` and `rust-analyzer`).

Note that this is only the default toolchain, you can override it on the CLI with the `+toolchain` pattern or with the `RUSTUP_TOOLCHAIN` environment variable.